css/_vertical_filmstrip_overrides.scss

```diff
@@ -103,6 +103,20 @@
                 right: 150px;
             }
         }
+
+        &.with-filmstrip.without-remote-videos {
```

Comment:
It's difficult to guess by the naming only what those classes are handling. We should remember to add some docs.